.swiftpm/xcode/xcshareddata/xcschemes/BrowserServicesKit-Package.xcscheme

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
    LastUpgradeVersion = "1540"
-   version = "1.7">
+   version = "1.8">
```

Comment:
It might worth checking with the team before upgrading this (and `SubscriptionTests.xcscheme` below)

Reply:
Ah I'll revert before merging as this isn't really the place.